src/pstore/pstore.c

```diff
@@ -0,0 +1,407 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+/* Copyright © 2019 Oracle and/or its affiliates. */
```

Comment:
Is this necessary? We generally avoid explicit copyright lines, because they are meaningless (the copyright conventions don't require them for anything), and quickly become stale and `git blame` delivers much better information anyway.

Reply:
Well, my employer requested it. There are over 190 other Copyrights in the systemd code.

Reply:
Yeah, because I successfully removed them from the other 3167 files ;)

Reply:
Oracle is still in the process of embracing SPDX ,but is headed in that direction. @konradwilk can weigh-in with information in this regard, if needed.

Reply:
This isn't *directly* connected to SPDX. The rationale is a bit different. We removed the copyrgiht lines after swithng to SPDX, because once we removed the noisy license boilerplate, the noisy copyright boilerplate became much more visible ;) But strictly speaking, those changes are independent.